scripts/scil_screenshot_bundle.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+import os
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import load_tractogram
+from dipy.tracking.streamline import transform_streamlines
+from fury import actor
+import matplotlib.pyplot as plt
+import nibabel as nib
+import numpy as np
+from scipy.ndimage import map_coordinates
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.utils.image import register_image
+from scilpy.viz.screenshot import display_slices
+
+DESCRIPTION = """
+   Register bundle to a template for screenshots using a reference.
+   The template are in /mnt/braindata/Other/simple_template_viz/
```

Comment:
Should not use something from our own repository

Reply:
And don't forget to add no skull.

Reply:
We could put an MNI152 (with an wmparc) it is always useful for 3MB of space

Reply:
It will be in the singularity not in the scilpy repo. Please update DESCRIPTION